blog-service/2024-01-19-event-analytics.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Event Analytics (Search) 
```

Comment:
will these release notes go out to all customers ?

Reply:
Yes, this will be published under this https://help.sumologic.com/release-notes-service/ page.

Reply:
i don't think we would want to announce to all customers right now. we can skip the release notes

Reply:
Okay, so you only need the Beta doc right now?

Reply:
correct

Reply:
Sure, the PR is updated now with just the Beta doc. So when can we publish this PR?